cloudformation.yaml

```diff
@@ -46,12 +46,7 @@ Resources:
             - s3:Get*
             - s3:Delete*
             Resource:
-            - arn:aws:s3::*:content-api-config/*
             - arn:aws:s3::*:content-api-dist/*
-            - arn:aws:s3::*:fastly-logs-audio
-            - arn:aws:s3::*:fastly-logs-audio/*
-            - arn:aws:s3::*:acast-logs-audio
-            - arn:aws:s3::*:acast-logs-audio/*
```

Comment:
These can be removed because `acast-` and `fastly-logs-audio` are/were sub-objects of `gu-audio-logs` 👍